core/page-assets/loader.php

```diff
@@ -35,6 +35,11 @@ private function init_assets() {
 					'version' => ELEMENTOR_VERSION,
 					'dependencies' => [],
 				],
+				'swiper' => [
+					'src' => $this->get_css_assets_url( 'swiper',  Plugin::$instance->experiments->is_feature_active( 'e_swiper_latest' ) ? 'assets/lib/swiper/v8/css/' : 'assets/lib/swiper/css/' ),
```

Comment:
Please extract to a function, like it was done inside frontend.php, for a more readable code.